homeassistant/components/withings/coordinator.py

```diff
@@ -18,12 +20,13 @@
     aggregate_measurements,
 )
 
-from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryAuthFailed
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 from homeassistant.util import dt as dt_util
 
+if TYPE_CHECKING:
+    from . import WithingsConfigEntry
```

Comment:
I'd suggest to move the block after the last import. That's how it's usually done. Tbh I though ruff would even auto-format it that way but that doesn't seem to be the case 🤔

Reply:
I also don't like that it doesn't enforce a space after the block. Maybe we should propose a change to ruff and/or black